### PR TITLE
Add to_datetime wrapper and include test

### LINKED ISSUE #188: Add the `pd.to_datetime` method so that it is chainable
It would be nice to have this function being chainable.

### PR DESCRIPTION
**Pull request by jonnybazookatone**
This PR resolves #188 .

# PR Description

Please describe the changes proposed in the pull request:

- Make `pd.to_datetime` a chainable method
- Add tests

# Relevant Reviewers

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
Thanks @jonnybazookatone! Could you also add yourself to the `AUTHORS.rst`?

**Comment by ericmjl** [comment]
This is ready to merge. Will wait till everyboyd is back so we can celebrate!

File: janitor/functions.py [hunk 1]
```diff
@@ -2599,3 +2599,34 @@ def update_where(
...
+        import janitor
+        df = pd.DataFrame(...).to_datetime('col1', format='%Y%m%d')
+
+    :param df: A pandas DataFrame.
+    :param column: Column name.
+    :param str format: time format to convert to.
+    :returns: A pandas DataFrame.
+    """
+
+    df[column] = pd.to_datetime(df[column], format=format)
```
[the remark this answers is not in the document]
Comment (jonnybazookatone) [reply janitor/functions.py]: Sure, I can change it. There's no need for `args` though, just `kwargs`.

**Comment by ericmjl** [comment]
Once Azure passes, we're ready to merge.

**Pull request merged**